From e0f0549ac6766ab565dbd581098b96792a807727 Mon Sep 17 00:00:00 2001
From: Peter Oliver <git@mavit.org.uk>
Date: Wed, 17 Jul 2024 00:06:02 +0100
Subject: [PATCH 4/4] Try harder to stabalise dired-test-bug27243-02

* test/lisp/dired-tests.el (dired-test-bug27243-02): Don't rely on the
expected point, since columns will move if the size of the parent directory
changes order of magnitude.
---
 test/lisp/dired-tests.el | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/test/lisp/dired-tests.el b/test/lisp/dired-tests.el
index ea47043dfc5..6078e99e13c 100644
--- a/test/lisp/dired-tests.el
+++ b/test/lisp/dired-tests.el
@@ -190,7 +190,6 @@ dired-test-bug27243-02
   "Test for https://debbugs.gnu.org/cgi/bugreport.cgi?bug=27243#28 ."
   (ert-with-temp-directory test-dir
     (let ((dired-auto-revert-buffer t)
-          (dired-free-space nil)
           buffers)
       ;; On MS-Windows, get rid of 8+3 short names in test-dir, if the
       ;; corresponding long file names exist, otherwise such names trip
@@ -216,7 +215,9 @@ dired-test-bug27243-02
             ;; Sanity check: point should now be back on the subdirectory.
             (should (eq (point) pt1))
             (push (dired test-dir) buffers)
-            (should (eq (point) pt1)))
+            (should (equal (dired-file-name-at-point)
+                           (concat (file-name-as-directory test-dir)
+                                   (file-name-as-directory "test-subdir")))))
         (dolist (buf buffers)
           (when (buffer-live-p buf) (kill-buffer buf)))))))
 
-- 
2.45.2

